src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1228,6 +1228,32 @@ components:
             - Queued
             - Processing
           description: Denotes the current status of the job in the queue
+          jobs:
+          type: object
```

Comment:
There as slight issues here:
1. The `jobs:` should be indented left one level to be correct.
2. The `jobs` is an array in the response I receive https://github.com/fossology/fossology/blob/74d85d8f362fbe807d3ec67215b5960e7db1e3cb/src/www/ui/api/Models/Job.php#L107

It will be very helpful if the whole object can be created as a member of `components/schemas` and then refered inside the `Job` object as
```yaml
    Job:
      type: object
      properties:
        id:
...
          description: Denotes the current status of the job in the queue
        jobs:
          type: array
          items:
            $ref: '#/components/schemas/JobQueue'
    JobQueue:
      type: object
      description: Information about individual agents
      properties:
        queueId:
          type: integer
          description: ID of the queued child job
        agentName:
          type: string
          description: Name of the agent
        status:
          type: string
          enum:
            - Completed
            - Failed
            - Queued
            - Processing
          description: Denotes the current status of the child job queued
        startTime:
          type: string
          description: Time when the child job was queued
        endTime:
          type: string
          description: Time when the child job was completed, empty when status is Failed/Queued/Processing
        itemsProcessed:
          type: integer
          description: Number of items processed by the agent
```